src/confirmations_impl.cc

```diff
@@ -285,7 +319,7 @@ std::map<std::string, std::string> ConfirmationsImpl::GetCatalogIssuers()
 }
 
 bool ConfirmationsImpl::IsValidPublicKeyForCatalogIssues(
```

Comment:
Nitpick: Is that a typo in the method name (Issues vs. Issuers)?

Reply:
That is a typo 👍